Dockerfile.builder

```diff
@@ -27,7 +27,7 @@ RUN chmod 777 -R ${HOME}
 
 RUN apt-get update \
     && apt-get install --no-install-recommends -y -q \
-    curl wget git unzip bzip2 jq \
+    curl wget git unzip bzip2 jq expect \
```

Comment:
We probably should update node and probably kubectl at the same time so we don't have to replace the builder image twice. @zherman0 was looking at that.

Reply:
so that means, once this PR is merged? https://github.com/openshift/console/pull/7280

should I update kubectl in this PR to some version? or is/will be there a special PR for that?

Reply:
Sorry @suomiy, I missed this question. Basically I think #7306 and this should be combined. That way we only need to update the builder image once since there are several steps.